plugins/scaffolder/src/components/fields/RepoUrlPicker/AzureRepoPicker.tsx

```diff
@@ -55,15 +57,29 @@ export const AzureRepoPicker = (props: {
         error={rawErrors?.length > 0 && !organization}
       >
         {allowedOrganizations?.length ? (
-          <Select
-            native
-            label="Organization"
-            onChange={s =>
-              onChange({ organization: String(Array.isArray(s) ? s[0] : s) })
-            }
-            disabled={allowedOrganizations.length === 1}
-            selected={organization}
-            items={organizationItems}
+          <Autocomplete
+          aria-label="Organization"
+          options={organizationItems}
+          getOptionLabel={(organizations) => organizations.label || "error getting label"}
+          disabled={allowedOrganizations.length === 1}
+          data-testid="select"
+          freeSolo
+          onChange={selected =>
+            onChange({
+              organization: String(Array.isArray(selected) ? selected[0] : selected),
+            })
+          } 
+          tabIndex={0}
```

Comment:
Probably want to remove this from all the `Autocompletes` to as it affects accessibility?